packages/application-shell/src/test-utils.js

```diff
@@ -20,18 +21,53 @@ import { createReduxStore } from './configure-store';
 // location is better for it.
 afterEach(memoryAdapter.reset);
 
-// This function renders any component within the application context.
+// These default values get merged with the values provided by the test from
+// the call to "render"
+const defaultUser = {
+  id: 'user-id-1',
+  email: 'sheldon.cooper@caltech.edu',
+  firstName: 'Sheldon',
+  lastName: 'Cooper',
+  language: 'en',
+  timeZone: 'Etc/UTC',
+};
+
+const defaultEnvironment = {
+  frontendHost: 'localhost:3001',
+  mcApiUrl: 'https://mc-api.commercetools.com',
+  location: 'eu',
+  env: 'production',
```

Comment:
The default values don't matter if I'm not mistaken. All requests to the outside are stubbed - and the tokens used to make the requests aren't valid either. Is there a concern I'm missing?

I prefer having the test-setup close to the production setup so that the tests are closer to real conditions. I would even like to use a real value for `frontendHost` but I didn't know what that would look like. `mc.commercetools.com`?

Reply:
Silly argument for using production: we want to test how the user perceives it as similar as possible. He will be on a production environment. Even if this might not change anything :).